crates/interledger-store-memory/src/store.rs

```
@@ -138,6 +139,8 @@ impl AccountStore for InMemoryStore {
     }
 }
 
+impl PubStore for InMemoryStore {}
```

I don't think this store should implement this trait because it doesn't actually implement the notifications functionality

---

As I note above, we need this or else `cargo check` fails hard in the repo:
```
    Checking interledger v0.4.1-alpha.1 (/Users/bstriegel/Code/interledger-rs/crates/interledger)
error[E0277]: the trait bound `interledger_store_memory::store::InMemoryStore: interledger_stream::server::PubStore` is not satisfied
   --> crates/interledger/src/cli.rs:225:13
    |
225 |             StreamReceiverService::new(server_secret.clone(), store.clone(), outgoing_service);
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `interledger_stream::server::PubStore` is not implemented for `interledger_store_memory::store::InMemoryStore`
    |
    = note: required by `interledger_stream::server::StreamReceiverService::<S, O, A>::new`

[clipped: another hundred lines of errors]
```